Source/Core/DolphinQt/HotkeyScheduler.cpp

```diff
@@ -139,10 +139,22 @@ void HotkeyScheduler::Run()
 
   while (!m_stop_requested.IsSet())
   {
-    Common::SleepCurrentThread(1000 / 60);
+    const u64 freelook_update_freq = 200;
+    const u64 interval = SystemTimers::GetTicksPerSecond() / freelook_update_freq;
+    const u64 now = CoreTiming::GetTicks();
 
-    if (Core::GetState() == Core::State::Uninitialized || Core::GetState() == Core::State::Paused)
+    if (now - m_last_freelook_ticked > interval)
```

Comment:
Guessing it should be >= interval? e.g. if interval is 0, it should run every time (I know it's an impossible case).